todo_app/data/item.py

```diff
@@ -1,13 +1,26 @@
 from enum import Enum
 from typing import Optional
 
+
 class ItemStatus(Enum):
     NOT_STARTED = 'Not Started'
     IN_PROGRESS = 'In Progress'
     COMPLETE = 'Complete'
 
+    @staticmethod
+    def from_str(label):
+        if label in ('Not Started'):
+            return ItemStatus.NOT_STARTED
+        elif label in ('In Progress'):
+            return ItemStatus.IN_PROGRESS
+        elif label in ('Complete'):
+            return ItemStatus.COMPLETE
+        else:
+            raise NotImplementedError
```

Comment:
```suggestion
            raise NotImplementedError()
```
Throwing a class directly seems wrong, though I guess it works. Might also be worth providing an error message.